docs/enterprise-guide/sql-snippets.md

```diff
@@ -0,0 +1,79 @@
+3. Give your folder a name, and optionally **Add a description** and/or place the folder in an existing folder (the **Folder this should be in** option).
+
+![Create new folder modal](./images/sql-snippets/create-new-folder-modal.png)
+
+### Creating a new SQL snippet
+
+When creating a SQL snippet in the Enterprise Edition, you'll also see an additional option to add that snippet to an existing folder (the **Folder this should be in** option). 
+
+![Add a snippet enterprise modal](./images/sql-snippets/enterprise-add-snippet.png)
+
+The default location is the **Top folder**, which is the root folder for all snippets and folders. You can add a snippet to a folder at any time (or relocate a snippet to another folder, provided you have Edit permission to both folders).
+
+Note that snippet names must be unique; folders do not affect this requirement.
+
+## Permissions
+
+Administrators (and only administrators) can set snippet visibility and editability by placing snippets in **folders**, then granting groups one of three permission levels with respect to those folders. If you're familiar with [collection permissions](/docs/latest/administration-guide/06-collections.html#setting-permissions-for-collections), the functionality is similar. For more on how collection and folder permissions work together, see [how folder permissions work](#how-folder-permissions-work) below.
+
+### Changing permissions on a folder
+
+Administrators can set the permissions on a folder by clicking on the ellipsis (**...**) next to a folder, and selecting **Change permissions**.
+
+You can additionally change the currently selected folder by mousing over to the top of the Snippets sidebar, clicking on the ellipsis (**...**) to the left of the **+**, and selecting **Change permissions**. When at the **Top folder**, selecting the **...** at the top of the sidebar will give Administrators the option to set permissions for all snippets, folders, and sub-folders.
+
+When changing permissions on a folder that has sub-folders, you have an option to extend those permissions to that folder's sub-folders by toggling the **Also change sub-folders** setting.
+
+### Options for folder permissions
+
+![Change permissions](./images/sql-snippets/change-permissions.png)
+
+There are three options for changing snippet folder permissions:
+
+- **Edit access (green checkmark icon)**. The default setting. When a folder is created, all users can view, edit, and archive or unarchive the folder's snippets. 
+- **View access (yellow eye icon)**. Users in groups with view access can view snippets in the folder, but not edit or archive/unarchive them. They can, of course, copy snippet code and create new snippets, with no effect on the existing snippets.
+- **Revoke access (red X icon)**. Users in groups with neither edit nor view permissions to a snippet folder will not see that folder's snippets in the sidebar, nor will any snippets in that folder populate those users' searches.
```

Comment:
We could note that questions that utilize snippets will still run even if a user doesn't have folder permissions for snippets the question uses.